website/src/views/denv1.ts

```diff
@@ -0,0 +1,150 @@
+    public readonly organism = Organisms.denv1;
+    public readonly mainDateField: string;
+    public readonly earliestDate = earliestDate;
+    public readonly locationFields = GENSPECTRUM_LOCULUS_LOCATION_FIELDS;
+    public readonly lineageFilters: LineageFilterConfig[] = [
+        {
+            lapisField: LINEAGE_FIELD_NAME,
+            placeholderText: 'Clade',
+            filterType: 'text' as const,
+        },
+    ];
+    public readonly useAdvancedQuery = false;
+    public readonly baselineFilterConfigs: BaselineFilterConfig[] = [
+        ...getGenspectrumLoculusFilters({
+            dateRangeOptions: fineGrainedDefaultDateRangeOptions,
+            earliestDate,
+        }),
+    ];
+    public readonly hostField: string = GENSPECTRUM_LOCULUS_HOST_FIELD;
+    public readonly authorsField = LOCULUS_AUTHORS_FIELD;
+    public readonly authorAffiliationsField = LOCULUS_AUTHORS_AFFILIATIONS_FIELD;
+    public readonly additionalFilters: Record<string, string> | undefined;
+    public readonly dataOrigins = [dataOrigins.insdc];
+    public readonly accessionDownloadFields = INSDC_ACCESSION_DOWNLOAD_FILES;
+    public readonly predefinedVariants = [
+        {
+            lineages: { [LINEAGE_FIELD_NAME]: '1I_K.1.1' },
+        },
+        {
+            lineages: { [LINEAGE_FIELD_NAME]: '1V_D.1.1' },
+        },
+        {
+            lineages: { [LINEAGE_FIELD_NAME]: '1V_E.1' },
+        },
+    ];
+    public readonly mutationAnnotations: MutationAnnotation[] = [];
+
+    public get aggregatedVisualizations() {
+        return getGenSpectrumLoculusAggregatedVisualizations(this, {
+            sublineages: {
+                label: 'Clades',
+                fields: [LINEAGE_FIELD_NAME],
+            },
+        });
+    }
+
+    constructor(organismsConfig: OrganismsConfig) {
+        this.mainDateField = organismsConfig.denv1.lapis.mainDateField;
+        this.additionalFilters = organismsConfig.denv1.lapis.additionalFilters;
+    }
+}
+
+const defaultDatasetFilter: DatasetFilter = {
+    locationFilters: {},
+    textFilters: {},
+    dateFilters: {
+        [GENSPECTRUM_LOCULUS_MAIN_FILTER_DATE_COLUMN]: dateRangeOptionPresets().lastYear,
+    },
+    numberFilters: {},
+};
+
+export class Denv1AnalyzeSingleVariantView extends GenericSingleVariantView<Denv1Constants> {
+    constructor(organismsConfig: OrganismsConfig) {
+        super(new Denv1Constants(organismsConfig), makeDatasetAndVariantData(defaultDatasetFilter));
+    }
+}
+
+export class Denv1CompareSideBySideView extends BaseView<
+    CompareSideBySideData,
+    Denv1Constants,
+    CompareSideBySideStateHandler
+> {
+    constructor(organismsConfig: OrganismsConfig) {
+        const constants = new Denv1Constants(organismsConfig);
+        const defaultPageState = makeCompareSideBySideData(defaultDatasetFilter, [
+            {},
+            {
+                lineages: {
+                    [LINEAGE_FIELD_NAME]: 'A.D.5.2',
```

Comment:
This is probably not a valid lineage for denv (denv 2, 3 and 4 also have this lineage in this spot)

Reply:
Thanks, I fixed it!